main.py

```diff
@@ -1,13 +1,25 @@
-import pandas as pd
+import os  # noqa F401
 
+import pandas as pd
 from src.utils.hdfs_mods import hdfs_load_json as read_json
 
-# TODO: read from config
-folder_path = "/dapsen/workspace_zone/mbs-results/"
-file_name = "snapshot-202212-002-2156d36b-e61f-42f1-a0f1-61d1f8568b8e.json"
-file_path = folder_path + file_name
+from mbs_results.data_cleaning import filter_responses
+from mbs_results.inputs import load_config
+
+# os.chdir("/home/cdsw/monthly-business-survey-results/")
 
-snapshot = read_json(file_path)
+
+# TODO:
+# run modules
+# run imputation
+# deal with QA outputs
+# basic logging
+# kwargs in config
+
+config = load_config()
```

Comment:
Had to install ` research_and_development==1.0.0` to get main script to run. Getting error relating to json loading on L19.